lib/fluent/plugin/output.rb

```diff
@@ -1202,7 +1204,10 @@ def backup_chunk(chunk, using_secondary, delayed_commit)
 
       def check_slow_flush(start)
         elapsed_time = Fluent::Clock.now - start
+        # millsec precision
+        @counters_monitor.synchronize { @flush_time_count += (elapsed_time * 1000).to_i }
```

Comment:
How about `(elapsed_time * 1000).to_i`  calculation outside of synchronize?

Reply:
https://github.com/fluent/fluentd/pull/2450/commits/05d842e944f43d9f7610db4d7ed62ea3399273ff fixed